bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingStatusDetail.java

```diff
@@ -23,7 +23,8 @@
     CONFIGURATION_ERROR(7),
     BRIDGE_OFFLINE(8),
     FIRMWARE_UPDATING(9),
-    DUTY_CYCLE(10);
+    FIRMWARE_UPDATE_PENDING(10),
```

Comment:
> If such a device is in standby mode it does not start the update -> FIRMWARE_UPDATE_PENDING.

This might easily get into conflict with CONFIGURATION_PENDING, which is a status that often occurs in the same situation (a device on standby). We should make sure that there is always only one valid status.

Wouldn't it make much more sense to add the new status to FirmwareStatus ("defines all possible statuses for the Firmware of a Thing"? Something like FirmwareStatus.UPDATE_QUEUED?